### DISCUSSION TITLE
Spare 50A CTs

### DISCUSSION THREAD

**Discussion by Telk0** [emporia-vue-local/esphome#316] [General]
Any interesseret in buying My spare 50A CTs used for gen 2. Got 8. Can be shipped in Europe on buyers Bill.

https://shop.emporiaenergy.com/products/set-of-eight-50a-sensors-for-gen-2-vue

**Comment by seppelp** [comment]
How much are you asking for?

**Reply by Telk0** [reply]
Feel free to make an offer. Was thinking 30euro + shipping

**Reply by juavinas** [reply]
I can buy, contact me thru email: juozas_usas@yahoo.com

**Reply by seppelp** [reply]
I was asking first, if its still available i would buy it as well. s.github@lahntal.net
